coredb-operator/src/statefulset.rs

```diff
@@ -182,6 +201,13 @@ pub fn stateful_set_from_cdb(cdb: &CoreDB) -> StatefulSet {
                             docker_setup_env
                             docker_create_db_directories
 
+                            cp -r /tmp/pg_sharedir/* $(pg_config --sharedir)/
+                            cp -r /tmp/pg_pkglibdir/* $(pg_config --pkglibdir)/
+                            chown -R :postgres $(pg_config --sharedir)
+                            chmod -R 2775 $(pg_config --sharedir)
+                            chown -R :postgres $(pg_config --pkglibdir)
+                            chmod -R 2775 $(pg_config --pkglibdir)
```

Comment:
`root` is the user that owns it but the `postgres` group has ownership. My thought was that this would let `trunk` manage all the extensions, not just the ones that are installed at runtime. If thats not what we want, then we probably only need to give the `postgres` group write permissions to: `$sharedir/extension`, `$sharedir/bitcode` (if it exists), and `$pkglibdir` 

```
-rw-r--r-- 1 root     postgres  153 Apr  5 17:58 vector--0.3.1--0.3.2.sql
```

> what about deleting the left-overs, meaning files that are in the sharedir, pkglibdirs already but originated from previous copies to the PVC?

Yes good idea. I'll handle that in a follow-up PR.

Reply:
I'll address the recursive `chown` in that next PR too